modules/chapter1/pages/section1.adoc

```diff
@@ -1,3 +1,105 @@
-= Section 1
+= Introduction
+
+
+Red{nbsp}Hat OpenShift Data Science (RHODS) is a platform for data scientists, machine learning engineers, and operation teams to implement the full lifecycle of AI-based applications.
+
+With RHODS, uses can gather data, use the data to train and evaluate their AI models, and serve and monitor those models.
+
+Users can also create _Data Science Pipelines_ to implement and automate model workflows.
+
+
+== What is Artificial Intelligence?
```

Comment:
This section is not needed for quick course audience. We can reuse for the GLS course later on. Please remove